### DISCUSSION TITLE
Q3 errors

### DISCUSSION THREAD

**Discussion by ghost** [javascripteverywhere/api#17] [Q&A]
while cloning the repository, I got a bunch of npm warnings and errors even though I tried to use npm install with npm audit fix.

nevertheless, I went on to create the express server but after clicking node src/index.js .

I meet an error I don't know how to fix.
Error: Cannot find module 'express'

I tried installing express separately but still got the same error.
here is log for error:
![image](https://user-images.githubusercontent.com/93210759/146561415-da94e638-1751-4064-bdd1-fa63f59381e8.png)

**Comment by Raikyo777** [comment]
I think, express module not installed. Try to install it directly.
npm install express --save
P.S. Do it then terminal in notedly folder (the module installed in /node_modules)
